game/theater/controlpoint.py

```diff
@@ -813,7 +826,8 @@ def find_ground_objects_by_obj_name(
                 found.append(g)
         return found
 
-    def is_friendly(self, to_player: bool) -> bool:
+    # TODO: The below two functions need to be refactored for Neutral base support or we need a new function that is called before these two functions to determine if the base is neutral or not.
+    def is_friendly(self, to_player: bool) -> Optional[bool]:
```

Comment:
Return-type seems off here, it's still a bool to be returned, but `to_player` should become `Optional[bool]` if my understanding is correct, and more importantly, _if_ we want to continue with the approach of having player & neutral as split variables 😅